scripts/commands/getenmity.lua

```diff
@@ -0,0 +1,30 @@
+---------------------------------------------------------------------------------------------------
+-- func: getenmity
+-- desc: prints the target mob's current CE and VE towards you
+---------------------------------------------------------------------------------------------------
+require("scripts/globals/status")
+---------------------------------------------------------------------------------------------------
+
+cmdprops =
+{
+    permission = 2,
+    parameters = ""
+}
+
+function error(player, msg)
+    player:PrintToPlayer(msg)
+    player:PrintToPlayer("!getenmity")
+end
+
+function onTrigger(player)
+
```

Comment:
Extra whitespace lurking around in this file, please remove

Reply:
I think I've cleared up the whitespace you're referring to.